README.md

```diff
@@ -656,8 +656,14 @@ line-separated JSON output file showing all differences found between
 the leader and the local metadata databases, for raft sessions where
 the repd process is a follower.
 
-Example command:
+First determine the number of databases managed by metadata.  Run the command
+```
+docker exec -u scality -it scality-metadata-bucket-repd ls -l /databases
+```
+If you have one line of output, you can run the first example command.  If you have multiple lines of output you have to use that output in the second example command for the tool.
+
 
+Example command for a single SSD:
```

Comment:
I think we can get rid of the single-SSD procedure since the multiple SSD is more generic, i.e. it would also work for single SSD.